data/transition-sites/dclg_localdirect.yml

```diff
@@ -0,0 +1,9 @@
+---
+site: dclg_localdirect
+whitehall_slug: department-for-communities-and-local-government
+homepage: https://www.gov.uk/government/organisations/department-for-communities-and-local-government
```

Comment:
There's a [short url for DCLG](https://github.com/alphagov/transition-config/blob/master/data/transition-sites/dclg.yml#L6) which you could add as `homepage_furl`.